catalog/resource_lakehouse_monitor_test.go

```diff
@@ -0,0 +1,225 @@
+package catalog
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/experimental/mocks"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestLakehouseMonitorCornerCases(t *testing.T) {
+	qa.ResourceCornerCases(t, ResourceLakehouseMonitor())
+}
+
+func TestLakehouseMonitorCreateTimeseries(t *testing.T) {
+	qa.ResourceFixture{
+		MockWorkspaceClientFunc: func(w *mocks.MockWorkspaceClient) {
+			e := w.GetMockLakehouseMonitorsAPI().EXPECT()
+			e.Create(mock.Anything, catalog.CreateMonitor{
+				FullName:         "test_table",
+				OutputSchemaName: "output.schema",
+				AssetsDir:        "sample.dir",
+				TimeSeries: &catalog.MonitorTimeSeriesProfileType{
+					Granularities: []string{"1 day"},
+					TimestampCol:  "timestamp",
+				},
+			}).Return(&catalog.MonitorInfo{
+				AssetsDir:        "sample.dir",
+				OutputSchemaName: "output.schema",
+				TableName:        "test_table",
+				Status:           catalog.MonitorInfoStatusMonitorStatusActive,
+			}, nil)
+			e.Create(mock.Anything, catalog.CreateMonitor{
+				FullName:         "test_table",
+				OutputSchemaName: "output.schema",
+				AssetsDir:        "sample.dir",
+				TimeSeries: &catalog.MonitorTimeSeriesProfileType{
+					Granularities: []string{"1 day"},
+					TimestampCol:  "timestamp",
+				},
+			}).Return(&catalog.MonitorInfo{
+				AssetsDir:        "sample.dir",
+				OutputSchemaName: "output.schema",
+				TableName:        "test_table",
+				Status:           catalog.MonitorInfoStatusMonitorStatusActive,
+			}, nil)
```

Comment:
This is duplicate? Why do we need to call CreateMonitor twice? Maybe this is a bug in the unit-testing framework?

Reply:
removed it.